source/com.android.billingclient/billing/Additions/Additions.cs

```diff
@@ -90,59 +83,43 @@ public Task<ConsumeResult> ConsumeAsync(ConsumeParams consumeParams)
         }
 
         [Obsolete("Use QueryPurchaseHistoryAsync(QueryPurchaseHistoryParams) instead")]
+        [Obsolete("QueryPurchaseHistory method was removed in Billing Client v8.0.0. Use QueryPurchases instead.")]
         public Task<QueryPurchaseHistoryResult> QueryPurchaseHistoryAsync(string skuType)
```

Comment:
If this method is not meant to be used anymore, is there an async task version of `QueryPurchases`? Maybe a `QueryPurchasesAsync` method?

Did Google just rename the method and return type? Or is it a whole separate implemenation.

Please link to the docs where this changed. Make sure too add any migration docs links into the obsolete attribute.